cfdm/read_write/write.py

```diff
@@ -165,6 +166,22 @@ def write(
             ``'NETCDF4'`` files use the version 4 disk format (HDF5)
             and use the new features of the version 4 API.
 
+        mode: `str`, optional
```

Comment:
I think we need a bit of white space to make the table render correctly:

```python
        mode: `str`, optional
            Specify the mode of write access for the output file. One
            of:

            =======  =================================================
            *mode*   Description
            =======  =================================================
            ``'w'``  Open a new file for writing to. If it exists and
                     *overwrite* is True then the file is deleted
                     prior to being recreated.

            ``'a'``  Open an existing file for appending new
                     information to. The new information will be
                     incorporated whilst the original contents of the
                     file will be preserved.
            =======  =================================================

            By default the file is opened with write access mode
            ``'w'``.

```

Reply:
Another good spot, thanks. I'll add that in as part of the documentation commit. Generally once I've made and am happy with the docs changes I planned to go in and make sure they rendered correctly but it's good to know this in advance.